oat-sa/tao-core
# Changed height of boxes to min-height everywhere

TAO-3772

Related to https://github.com/oat-sa/extension-tao-nccer-delivery/pull/249, i.e. the fix can only be seen on NCCER after installing this fix

This work is originally done fro NCCER but it would be important to test it on another project too. 
Screens to test on NCCER:
- _Select Your Testing Location_ screen
- any of the subsequent screens

Generic screen on any other project:
- Delivery entry screen

Instructions:
- Inspect the text on any of the boxes
- Replace the text within the `<p/>` with a bunch of _Lorem Ipsum_
- On the generic screen pay special attention to the link at the bottom right of each grey box (which is absent in NCCER)

Box should now adapt to height rather than being overflown
